plugin.js

```diff
@@ -21,6 +21,7 @@ const MODULE_EXPORTS = 'module.exports = ';
 const USE_STRICT = '\'use strict\'; ';
 
 const modifiedFiles = new Set();
+const addedPrefixes = new Map();
```

Comment:
This map obsoletes `modifiedFiles` set, as the keys of the former are the values of the latter. How about turning `modifiedFiles` into a map instead, with values of shape `{ prefix: '...', suffix: '...' }`?